source/s3_auto_ranged_put.c

```diff
@@ -102,6 +102,34 @@ static int s_s3_auto_ranged_put_pause(
     struct aws_s3_meta_request *meta_request,
     struct aws_s3_meta_request_resume_token **resume_token);
 
+/**
+ * Helper to compute request body size.
+ * Basically returns either part size or if content is not equally divisible into parts, the size of the remaining last
+ * part.
+ */
+static size_t s_compute_request_body_size(
```

Comment:
Trivial: Why are we moving this function around? I don't see any change made?

Reply:
moved it back. logic was simplified and i no longer need it at the start